### DISCUSSION TITLE
There's an error. What kind of question is that?

### DISCUSSION THREAD

**Discussion by mu116699** [IDEA-Research/Grounded-Segment-Anything#133] [Q&A]
try:
    from groundingdino import _C
except:
    warnings.warn("Failed to load custom C++ ops. Running on CPU mode Only!")

**Comment by mu116699** [comment]
using cpu

**Comment by rentainhe** [comment]
@SlongLiu

**Reply by rentainhe** [reply]
Maybe you should set `CUDA_HOME` before installing GroundingDINO as this issue mentioned: https://github.com/IDEA-Research/Grounded-Segment-Anything/issues/136
